dbgpt/core/interface/knowledge.py

```diff
@@ -132,3 +133,11 @@ def chunk2llamaindex(cls, chunk):
                 "Please install llama_index by command `pip install llama_index"
             )
         return TextNode(text=chunk.content, metadata=chunk.metadata)
+
+
+class LoadedChunk(Chunk):
```

Comment:
Should `LoadedChunk` be changed to `ParentChunk`? This class should be placed inside `load_doc_graph` for better results.